### DISCUSSION TITLE
ReactJS

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#33550] [Other Feature Feedback, Questions, & Ideas]
Hey everyone,

Okay, so currently I have started BackEnd and want to explore that field to a whole new level. Although using frontEnd I created my own website but that is just half of my portfolio. To make the website responsive, I need to learn ReactJS so that I can many different pages and make them look beautiful, responsive, and meaningful.

I request you community please help me through this and see how I contribute to the BackEnd ✌🏻

**Comment by ossanna16** [comment]
Hi @iLakshya and welcome to our community! What kind of help are you specifically looking for? Maybe I can help 😸 

Also to get started, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121) and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912).
